package/base-files/files/bin/board_detect

```diff
@@ -1,14 +1,22 @@
 #!/bin/sh
 
-CFG=$1
+REAL_CFG=$1
 
-[ -n "$CFG" ] || CFG=/etc/board.json
+[ -n "$REAL_CFG" ] || REAL_CFG=/etc/board.json
 
-if [ -d "/etc/board.d/" ] && [ ! -s "$CFG" ]; then
+if [ -d "/etc/board.d/" -a ! -s "$REAL_CFG" ]; then
```

Comment:
You revert the change you did in the previous commit again 
```
if [ -d "/etc/board.d/" ] && [ ! -s "$REAL_CFG" ]; then
```

Reply:
Very good catch. Fixed.